database/migration/src/main/resources/changesets/call_caching_job_detritus.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="rmunshi" id="call_caching_job_detritus">
+        <comment>
+            One row per standard output file per job (i.e, one row for the stdout, stderr, etc)
+            The standard output key contains the file extension type, such as stdout, and the standard
+            output value contains the file path, such as gs://cromwell-root/hello/call-hello/hello-stdout.log.
+        </comment>
+        <createTable tableName="CALL_CACHING_JOB_DETRITUS">
+            <column autoIncrement="true" name="CALL_CACHING_JOB_DETRITUS_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_CALL_CACHING_JOB_DETRITUS_ID"/>
+            </column>
+            <column name="JOB_DETRITUS_KEY" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="JOB_DETRITUS_VALUE" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="RESULT_METAINFO_ID" type="INT">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <!-- Don't want to store the same standard output key twice for each job/RESULT_METAINFO_ID) -->
```

Comment:
No, let's just keep it.  Part of my issue was being confused with the "standard output files" language.